### PR TITLE
Semantic UI Daterangepicker

### PR DESCRIPTION
**Pull request by martykan**
Like the Bootstrap Daterangepicker, but for Semantic UI.

### CODE REVIEW

File: files/semantic-ui.daterangepicker/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "semantic-ui.daterangepicker",
+  "repo": "BreadMaker/semantic-ui-daterangepicker",
+  "files": {
+    "include": ["daterangepicker.css", "daterangepicker.js", "daterangepicker.min.css", "daterangepicker.min.js"]
```
Comment (jimaek) [inline files/semantic-ui.daterangepicker/update.json]: Most of these files are not here https://github.com/BreadMaker/semantic-ui-daterangepicker/tree/v1.3.21

Comment (martykan) [inline files/semantic-ui.daterangepicker/update.json]: The older versions use a different naming scheme.

Comment (jimaek) [inline files/semantic-ui.daterangepicker/update.json]: 1.3.21 is the latest version according to this https://github.com/BreadMaker/semantic-ui-daterangepicker/releases

**Comment by martykan** [comment]
The releases seem to be of the Bootstrap version. I will close this request since the project needs to get the releases sorted out, before it can be published here.

**Pull request closed without merging**